README.md

```diff
@@ -294,6 +294,10 @@ $(Specified Directory)
 
 1. **Replace version codes** *(String, Required)* - You may specify which APK version codes should be replaced in the track with this deployment. Available options are: *All*, *List* - comma separated list of version codes, *Regular expression* - a regular expression pattern to select a list of APK version codes to be removed from the track with this deployment, e.g. _.\\*12?(3|4)?5_ 
 
+2. **Sent changes to review** *(Boolean, Optional)* - If you are getting the next error `Changes cannot be sent for review automatically. Please set the query parameter changesNotSentForReview to true`, select this option.
```

Comment:
Could you please elaborate more a little bit here - not only from the point of resolving of error, but from a point of purpose of this fastlane paremeter?

Reply:
@anatolybolshakov updated it.